### PR TITLE
Automatically generates core weekly posts 32 to 36

### PR DESCRIPTION
**Pull request by matks**

### CODE REVIEW

File: news/_posts/core-weekly/2019-09-01-coreweekly-week-35-2019.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+
+* [#37](https://github.com/PrestaShop/ps_emailsubscription/pull/37): Allow to have a custom template when rendering in a column, by [@Progi1984](https://github.com/Progi1984)
+
+### Watermark module
+
+* [#11](https://github.com/PrestaShop/watermark/pull/11): Remove "FollowSymLinks". Thank you [@antondollmaier](https://github.com/antondollmaier)
+
+<hr />
+
+Thank you to the contributors whose pull requests were merged since the last Core Weekly Report: @jf-viguier, @PierreRambaud, @jolelievre, @Progi1984, @matthieu-rolland, @nesrineabdmouleh, @davidglezz, @boubkerbribri, @matrixino, @PeeyushAgrawalWebkul, @pojebunny, @Amazzing, @zuk3975, @202-ecommerce, @sarjon, @antondollmaier!
```
Comment (ghost) [inline news/_posts/core-weekly/2019-09-01-coreweekly-week-35-2019.md:131]: I think the "!" brakes the link to @ antondollmaier 
(last name in the list)

Comment (ghost) [reply news/_posts/core-weekly/2019-09-01-coreweekly-week-35-2019.md:131]: Happens every time :)

Comment (ghost) [reply news/_posts/core-weekly/2019-09-01-coreweekly-week-35-2019.md:131]: oh ok, doesn't seem to be a link actually. All good 👍 
Unless you think it's better to make it a link, your call @matks & @ttoine

**Comment by ttoine** [comment]
There is an issue in the header of your articles. In the preview of your generated articles, this look like this:
Selection_489

It should be like that:
Selection_490

File: news/_posts/core-weekly/2019-09-01-coreweekly-week-35-2019.md (hunk 1 above)
Comment (ttoine) [reply news/_posts/core-weekly/2019-09-01-coreweekly-week-35-2019.md:131]: a link will be created by Jekyll when the page will be generated from the markdown.
usually, the ! is not a problem at all.
note that it is not mandatory to add a complete markdown link when using @nickname with github pages. they will be converted.

**Comment by matks** [comment]
> There is an issue in the header of your articles. In the preview of your generated articles, this look like this:

It was because I added a blank line at the top of the file. It's fixed now.

**Comment by matks** [comment]
Bugfix submitted for the generator: https://github.com/PrestaShop/core-weekly-generator/pull/4/

**Comment by matks** [comment]
@ttoine Shall we publish ? 😉

**Comment by ttoine** [comment]
should we let the current date of publication and publish them all the same day ?
or, do we put the date of the monday after each week number ?

**Comment by matks** [comment]
> or, do we put the date of the monday after each week number ?

That would be lying 😅 

> should we let the current date of publication and publish them all the same day ?

If you're OK to publish, I change the date and publish them all today

**Comment by ttoine** [comment]
Then do what you think is the best and publish them today 👍

**Comment by matks** [comment]
Publishing now 👍

**Pull request merged**